docs/guide/docs/developers/tutorials.md

```diff
@@ -67,10 +82,65 @@ window.botpressWebChat.init({
   showConversationsButton: true, // Whether or not to show the conversations button
   showUserName: false, // Whether or not to show the user's name
   showUserAvatar: false, // Whether or not to show the user's avatar
-  enableTranscriptDownload: false // Whether or not to show the transcript download button
+  enableTranscriptDownload: false, // Whether or not to show the transcript download button
+  externalAuthToken: 'my jwt token', // Defines a token that is sent with each messages to Botpress
+  userId: null // Allows you to override the default user id. Make sure it is not possible to guess it!
 })
 ```
 
+#### Configuring the Webchat during a conversation
+
+The method `window.botpressWebChat.configure` allows you to change the configuration of the chat during a conversation without having to reload the page
+
+## Connecting your bot with your existing backend
+
+Botpress makes it easy to communicate data securely between your backend and your bot using JWT. Store any data you'd like in the token and pass it to the webchat using the `externalAuthToken` configuration option.
+
+If the token is valid, the content will be available under `event.credentials` when processing the event. If there is no token or if it is invalid, `credentials` will stay undefined.
+
+How to enable:
+
+1. Configure `pro.externalAuth` in the file `botpress.config.json`
```

Comment:
Bugs me also, but it could be used by channel web, converse, and any other channels that supports custom headers. So, probably it could be called externalChatAuth, but this is really about authenticating the user with the business' existing backend, not purely 'chat'. In the business owner's perspective, the chat user can be 'Botpress Authenticated' (studio/admin) and 'External authenticated' (meaning their own auth)